website/public/js/controllers/groupsCtrl.js

```diff
@@ -224,6 +224,9 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
       if ($scope.query === undefined || $scope.query === null) {
         return false;
       }
+      if (msg.username == undefined) {
+        return false;
```

Comment:
Is there any instance where we would want `$scope.query === false`? If not, how simplifying it to:
```
if (!$scope.query || !msg.username) {
    return false;
}
```